spec/support/ansible_shared/automation_manager/credential.rb

```diff
@@ -79,6 +79,11 @@
       )
     end
 
+    it ".create_in_provider_queue to fail with incompatible manager" do
+      wrong_manager = FactoryGirl.create(:provider_foreman).managers.first
```

Comment:
You only need a single manager that is the wrong type, not a Provider, Manager and Authentication.  Avoid creating extra database records and looking them up.  Prefer the `:ext_management_system` or `:automation_manager` factory.

Reply:
Right! Done